modules/perception/inference/tensorrt/rt_utils.cc

```diff
@@ -28,39 +28,50 @@ bool ReadProtoFromTextFile(const std::string &filename,
                            google::protobuf::Message *proto) {
   int fd = open(filename.c_str(), O_RDONLY);
   if (fd < 0) {
-    AERROR << "cannot open file " << filename;
+    AERROR << "Failed to open file: " << filename;
     return false;
   }
   google::protobuf::io::FileInputStream raw_input(fd);
+  raw_input.setCloseOnDelete(true);
 
-  bool success = google::protobuf::TextFormat::Parse(&raw_input, proto);
+  if (!google::protobuf::TextFormat::Parse(&raw_input, proto)) {
+    AERROR << "Failed to parse proto file: " << filename;
+    return false;
```

Comment:
The close(fd) is not run if returned here.

Reply:
Umm...thank you for your review. those three methods all should be close the file handler at first before return in case of parsing failure. will fix it soon.